app/views/app_configs/_form.erb

```diff
@@ -65,7 +65,10 @@
       Configure notifications that will be sent to users.
     </p>
     <p>
-      Emails make use of variables; variables are placeholders that are automatically replaced with user- and reservation- specific information.<br />
+      Please note that by default all emails sent to users include a salutation and a closing.
```

Comment:
Nice, let's just clarify *what* a salutation and a closing are in case someone doesn't know, e.g. `... a salutation (e.g. "Dear John") and a closing (e.g. "Sincerely, John Doe Library").` or something like that.